sdk-python/create_bulk_messages.py

```diff
@@ -0,0 +1,76 @@
+import argparse
+import os
+import uuid
+import pandas as pd
+from awaazde.constants import CommonConstants
+from awaazde.models import Message
+from awaazde.utils import ADMessageUtils
+
+message_data = ''
+
+
+def parseArguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("username", help="Username", type=str)
+    parser.add_argument("password", help="Password", type=str)
+    parser.add_argument("organization", help="Organization ", type=str)
+    parser.add_argument("csv_file_path", help="Path for the csv ", type=str)
+    args = parser.parse_args()
+    return args
+
+
+def pick_messages_from_file(csvFilePath):
+    df = pd.read_csv(csvFilePath, encoding="ISO-8859-1")
+    message_data = df.replace({pd.np.nan: None})
+    headers = df.head()
+    message_data = message_data.to_dict('records')
+    message_data = Message(message_data)
+    return headers, message_data
+
+
+def get_last_tag(headers):
+    tags = [x for x in headers if x.startswith(CommonConstants.TAG_FIELD)]
+    tags = sorted(tags, reverse=True)
+    if len(tags) > 0:
+        p = int(filter(str.isdigit, str(tags[0])))
+        return 'tag{}'.format(p + 1)
+    else:
+        return CommonConstants.TAGS_FIELD
+
+
+def schedule_calls(csv_file_path):
```

Comment:
This naming seems out of nowhere. Unable to follow it as it relates to the name of this command line app

Reply:
Why it's taking a CSV path is also unclear, based on the given name

Reply:
@ashwini-ad not addressed